Lib/test/test_largefile.py

```diff
@@ -168,7 +168,7 @@ def test_it(self):
 @unittest.skipIf(not hasattr(os, 'sendfile'), 'sendfile not supported')
 class TestSocketSendfile(LargeFileTest, unittest.TestCase):
     open = staticmethod(io.open)
-    timeout = 3
+    timeout = 20
```

Comment:
I suggest to use support.SHORT_TIMEOUT, or event support.LONG_TIMEOUT. See the new timeout constants:
https://docs.python.org/dev/library/test.html#test.support.LOOPBACK_TIMEOUT

Reply:
Good idea. Done.